_data/experience.yml

```diff
@@ -31,13 +31,13 @@
   #quote: >
   # Short description of the company (optional)
   description: | # this will include new lines to allow paragraphs
-    Developed a data driven rendering library and revamped the MoPho app’s UX leading to an [acquisition by Shutterfly in 2012](https://techcrunch.com/2012/09/17/shutterfly-acquires-penguin-digital-makers-of-mobile-photo-print-shop-app-mopho/). Revamped UX and app architecture of the Shutterfly branded apps as well as introduced new features (store, collage, photo filters). Led development of the Tiny Prints app for creating, editing, and purchasing photo products with a native iOS experience. Awarded US Patent 9,153,056, [Adaptive and fast image collage creation](https://patents.google.com/patent/US20140307980).
-    
+    Developed a data-driven rendering library and revamped the MoPho app’s UX, leading to an [acquisition by Shutterfly in 2012](https://techcrunch.com/2012/09/17/shutterfly-acquires-penguin-digital-makers-of-mobile-photo-print-shop-app-mopho/). Revamped UX and app architecture of the Shutterfly branded apps as well as introduced new features (store, collage, photo filters). Led development of the Tiny Prints app for creating, editing, and purchasing photo products with a native iOS experience. Awarded US Patent 9,153,056, [Adaptive and fast image collage creation](https://patents.google.com/patent/US20140307980).
```

Comment:
I am passionate about dashes in [phrasal adjectives](https://grammarist.com/grammar/phrasal-adjectives/).

Reply:
👍